zuul.d/pods.yaml

```diff
@@ -1,10 +1,40 @@
 ---
 - job:
-    name: cifmw-pod-pre-commit
+    name: cifmw-pod-base
     nodeset:
       nodes:
         - name: container
           label: pod-centos-9-stream
     description: |
-      Run pre-commit against ci-framework
-    run: ci/playbooks/pre-commit.yml
+      Run lightweight jobs in pods
+    run: ci/playbooks/pod-jobs.yml
+
+- job:
+    name: cifmw-pod-pre-commit
+    parent: cifmw-pod-base
+    vars:
+      run_test: "pre_commit_nodeps"
+
+- job:
+    name: cifmw-pod-ansible-test
+    parent: cifmw-pod-base
+    vars:
+      run_test: "ansible_test_nodeps"
+
+- job:
+    name: cifmw-pod-zuul-files
+    parent: cifmw-pod-base
+    vars:
+      run_test: "check_zuul_files"
+    files:
+      - ^zuul.d/.*
+      - ^ci/templates/.*
+      - ^ci/config/.*
+
+- job:
+    name: cifmw-pod-k8s-snippets-source
+    parent: cifmw-pod-base
+    vars:
+      run_test: "check_k8s_snippets_comment"
+    files:
+      - ^roles/ci_gen_kustomize_values/templates/.*
```

Comment:
Why did it run in this PR? I wasn't expecting this job to report in this PR.

Reply:
hmm good question. Not that it takes too long, but.. indeed, it shouldn't run.

I can investigate this in a follow-up. The job itself is a matter of 5-6 minutes max, not taking too much time... But that whole patch is blocking further job on saving resources with actual nodeset :(.